piwik-sdk/src/main/java/org/piwik/sdk/DownloadTracker.java

```diff
@@ -50,20 +68,26 @@ public DownloadTracker(Tracker tracker, TrackMe baseTrackMe) {
         mContext = piwik.getContext();
         mPackageName = mContext.getPackageName();
         mPackMan = mContext.getPackageManager();
+        try {
+            mPkgInfo = mPackMan.getPackageInfo(mPackageName, 0);
+            mName = mPkgInfo.packageName;
+            mVersion = getVersion();
+            if (mVersion == null){
```

Comment:
If `getPackageInfo` did not throw an exception then `getVersion()` is never null. A packageInfo object always has a `versionCode`. The null check and extra exception is superfluos.